components/admin-dashboard/whitelisted-addresses.ts

```diff
@@ -1,4 +1,3 @@
-export const whitelistedAddresses = process.env
-  .NEXT_PUBLIC_WHITELISTED_ADDRESSES
-  ? process.env.NEXT_PUBLIC_WHITELISTED_ADDRESSES.split(",")
-  : [];
+import { env } from "../../config/env.mjs";
```

Comment:
use the consistent import, the `@/config/env.mjs`

Reply:
I have updated this!